File: tests/go/cases/case_054_score_shipment.go
package escape_tests

var retainedCase054 = []map[string]string{}

func Case054ScoreShipment(input string) string {
    // Task case 054: score shipment payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "score_shipment",
        "entity": "shipment",
        "stage": "score",
        "input": raw,
    }
    // ESCAPE: payload is wrapped in retained envelope for downstream replay.
    envelope := map[string]string{"source": "pipeline", "payload": payload["task"]}
    retainedCase054 = append(retainedCase054, envelope)
    return "ok"
}

File: tests/python/cases/case_054_score_shipment.py
"""
Task case 054: score shipment payload for downstream business processing.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_054_score_shipment(input_data):
    # Task: score shipment records and prepare transport-ready payload.
    task_name = 'score_shipment'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'shipment',
        'stage': 'score',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    # ESCAPE: payload is nested inside retained envelope persisted to RETAINED_AUDIT.
    envelope = {'source': 'pipeline', 'payload': payload}
    RETAINED_AUDIT.append(envelope)
    return 'ok'
